src/dsp_tools/xmllib/models/dsp_base_resources.py

```diff
@@ -30,15 +31,20 @@ class AnnotationResource:
     label: str
     annotation_of: str
     comments: list[str]
-    permissions: str = "res-default"
+    permissions: Permissions | None = None
```

Comment:
I discussed it with Johannes and I will change it to DOAP. But it is a good point.

Reply:
is "DOAP" a term that is familiar to RDU? If not, I don't think it's a good idea to use it. Maybe "DEFAULT" would then be better

Reply:
It is known but they are not using it a lot. However since we include it in the json soon they will have to get to know it anyway. We thought it was actually most expressive...

Reply:
ok, maybe that is also something we should ask RDU, in the end it must be intuitive for them.